contrib/release-tools/release.sh

```diff
@@ -23,6 +24,9 @@ function common_finalize {
 
 trap common_finalize EXIT
 
+git remote |grep "^upstream\$" -q || error_out code=127 message="There must be a git remote named 'upstream' available. The user running this script must have privilege to push commits to the 'upstream' remote"
```

Comment:
```suggestion
git remote get-url upstream >/dev/null || error_out code=127 message="There must be a git remote named 'upstream' available. The user running this script must have privilege to push commits to the 'upstream' remote"
```
Optional. A little more direct.

Reply:
Ah, I didn't know about that. In that case, we can be more strict here and verify that it contains the string `fedora-modularity/libmodulemd.git` as well, to be sure that the remote is correct.